romanisim/image.py

```diff
@@ -696,8 +701,7 @@ def simulate(metadata, objlist,
     else:
         l1, l1dq = romanisim.l1.make_l1(
             counts, ma_table_number, read_noise=read_noise, rng=rng, gain=gain,
-            crparam=crparam,
-            linearity=linearity, tstart=image_mod.meta.exposure.start_time,
+            crparam=crparam, linearity=inverse_linearity, tstart=image_mod.meta.exposure.start_time,
```

Comment:
Let's change the name here; linearity = inverse_linearity is asking for trouble..

Reply:
I changed it here and throughout l1 for clarity.